tests/acceptance/features/apiSharingNg/shareInvitations.feature

```diff
@@ -1208,3 +1208,53 @@ Feature: Send a sharing invitations
       | Viewer           | folder        | FolderToShare  |
       | Editor           | folder        | FolderToShare  |
       | Uploader         | folder        | FolderToShare  |
+
+
+  Scenario Outline: send share invitation to wrong user id with different roles
+    Given user "Alice" has uploaded file with content "to share" to "/textfile1.txt"
+    And user "Alice" has created folder "FolderToShare"
+    When user "Alice" tries to send the following share invitation using the Graph API:
+      | resourceType    | <resource-type>                      |
+      | resource        | <path>                               |
+      | space           | Personal                             |
+      | shareeId        | a4c0c83e-ae24-4870-93c3-fcaf2a2228f7 |
+      | shareType       | user                                 |
+      | permissionsRole | <permissions-role>                   |
+    Then the HTTP status code should be "400"
+    And the JSON data of the response should match
+      """
+      {
+        "type": "object",
+        "required": [
+          "error"
+        ],
+        "properties": {
+          "error": {
+            "type": "object",
+            "required": [
+              "code",
+              "message"
+            ],
+            "properties": {
+              "code": {
+                "type": "string",
+                "pattern": "generalException"
+              },
+              "message": {
+                "type": "string",
+                "enum": [
+                  "itemNotFound: not found"
+                ]
+              }
+            }
+          }
+        }
+      }
+      """
+    Examples:
+      | permissions-role | resource-type | path           |
+      | Viewer           | file          | /textfile1.txt |
+      | File Editor      | file          | /textfile1.txt |
+      | Viewer           | folder        | FolderToShare  |
+      | Editor           | folder        | FolderToShare  |
+      | Uploader         | folder        | FolderToShare  |
```

Comment:
maybe we can reduce the roles for negative test scenarios.

Reply:
I have used the `Viewer` role only for negative cases for sending share invitations for resources.